aquarius/run.py

```diff
@@ -97,7 +97,12 @@ def start_events_monitor():
     rpc = os.environ.get("EVENTS_RPC", "")
     Web3Provider.init_web3(provider=get_web3_connection_provider(rpc))
     ContractHandler.set_artifacts_path(get_artifacts_path())
-    if get_network_name().lower() == "rinkeby":
+    try:
+        use_poa_middleware = bool(int(os.getenv("USE_POA_MIDDLEWARE", "0")))
```

Comment:
the only comment I have is that it seems we are using ints as booleans for more variables. Could we have a function for that? e.g. get_bool_from_env(key)

It would reduce the number of separate try/except blocks